README.ja.md

```
@@ -1,27 +1,27 @@
 
-Iris は Slack ワークスペースと、ローカルで動く [Claude Code](https://claude.com/claude-code) CLI をつなぎます。Slack のスレッドから Claude に話しかけると、Iris が Claude Code を常駐プロセスとして起動し、出力をストリームで返し、ツール実行の許可をクリック可能なボタンとして提示します。
+Iris は Slack ワークスペースと、ローカルで動く **エージェント CLI**（[Claude Code](https://claude.com/claude-code) ・ [**Pi**](https://github.com/earendil-works/pi) ・ **Hermes**）をつなぎます。Slack のスレッドからエージェントに話しかけると、Iris がそのエージェントを常駐プロセスとして起動し、出力をストリームで返し、ツール実行の許可をクリック可能なボタンとして提示します。
 
-これは [cc-connect](https://github.com/chenhg5/cc-connect)（14 エージェント × 13 プラットフォーム対応の汎用ブリッジ）の設計思想だけを参考に、**Slack + Claude Code の組み合わせだけ**に絞って意図的に小さく作り直したものです。プラグインレジストリ・マルチプラットフォーム/エージェントの抽象・provider 切替・cron・relay・TTS などは持ちません。約 700 行（cc-connect の約 1/100）。
+これは [cc-connect](https://github.com/chenhg5/cc-connect)（14 エージェント × 13 プラットフォーム対応の汎用ブリッジ）の設計思想だけを参考に、**Slack ＋ 選択可能なエージェント・バックエンド（Claude Code / Pi / Hermes）**に絞って意図的に小さく作り直したものです。プラグインレジストリ・マルチプラットフォームの抽象・provider 切替・cron・relay・TTS などは持ちません。約 700 行（cc-connect の約 1/100）。
 
 > 虹の女神 **Iris**（神々と人間をつなぐ伝令）に由来。Hermes / Mnemosyne / Argus と同じ神話系の命名。
 
 ## しくみ
 
 ```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)